app/utils/Filters.scala

```diff
@@ -1,15 +0,0 @@
-package utils
-
-import javax.inject.Inject
-
-import play.api.http.HttpFilters
-import play.api.mvc.EssentialFilter
-import play.filters.csrf.CSRFFilter
-import play.filters.headers.SecurityHeadersFilter
-
-/**
- * Provides filters.
- */
-class Filters @Inject() (csrfFilter: CSRFFilter, securityHeadersFilter: SecurityHeadersFilter) extends HttpFilters {
-  override def filters: Seq[EssentialFilter] = Seq(csrfFilter, securityHeadersFilter)
```

Comment:
Is the `SecurityHeadersFilter` obsolete with the new `CSPFilter`. The migration guide for 2.7 says only that the `play.filters.headers.contentSecurityPolicy` is obsolete and `CSPFilter` should be used instead. But I think the `SecurityHeadersFilter` sets also other security based headers. What is with the `CSRFFilter`?

So if this class will be removed, the other filters should be added in the config!?

Reply:
The SecurityHeadersFilter is fine -- it's the content security policy property that got deprecated because it didn't adequately capture the complexity of CSP https://www.playframework.com/documentation/2.7.0/Migration27#play.filters.headers.contentSecurityPolicy-deprecated-for-CSPFilter.

The CSRFFilter is enabled as part of the default filters https://www.playframework.com/documentation/2.6.x/Migration26#Default-Filters